lms/djangoapps/verify_student/views.py

```diff
@@ -1024,12 +1024,6 @@ def results_callback(request):
             "Result {} not understood. Known results: PASS, FAIL, SYSTEM FAIL".format(result)
         )
 
-    # If this is a reverification, log an event
-    if attempt.window:
-        course_id = attempt.window.course_id
-        course_enrollment = CourseEnrollment.get_or_create_enrollment(attempt.user, course_id)
-        course_enrollment.emit_event(EVENT_NAME_USER_REVERIFICATION_REVIEWED_BY_SOFTWARESECURE)
```

Comment:
Please also delete the constant `EVENT_NAME_USER_REVERIFICATION_REVIEWED_BY_SOFTWARESECURE`